stories/base/Typography.stories.tsx

```diff
@@ -50,6 +51,51 @@ const orderedlList = [
   { id: "2", text: "Ordered list text goes here" },
   { id: "3", text: "Ordered list text goes here" },
 ];
+const tableContents = (
+  <>
+    <thead>
+      <tr>
+        <th align="left">Bank to Dwolla Network Clearing</th>
```

Comment:
I don't believe alignment is something that you can specify in markdown (correct me if I'm wrong!), so should we left align the header cells by default?

Reply:
You should be able to specify alignment in tables with markdown. Example below

| Default Header | Left Align | Right Align | Center Align |
| --- | :-- | --: | :-: |

Reply:
The More You Know! Thanks Spencer! Disregard this comment then